ding/framework/supervisor.py

```diff
@@ -0,0 +1,311 @@
+from copy import copy
+import multiprocessing as mp
+import threading
+import queue
+import platform
+from time import sleep
+import traceback
+import uuid
+from ditk import logging
+from dataclasses import dataclass, field
+from typing import Any, Callable, Dict, List, Optional, Union
+from enum import Enum
+
+
+@dataclass
+class SendPayload:
+    proc_id: int
+    req_id: str = field(default_factory=lambda: uuid.uuid1().hex)
```

Comment:
uuid4 may be better?

Reply:
The difference between uuid1 and uuid4 is whether or not a timestamp is used (resulting in ordered ids), where the order of ids is meaningful, so uuid1 is more appropriate.

Reply:
ok, add some comments about it

Reply:
done